CHANGELOG.md

```diff
@@ -16,7 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org).
   This means `zebrad` no longer contains users' viewing keys.
 - We're no longer using general conditional compilation attributes for `tor`,
   but only feature flags instead.
-- Fixed a bug with trailing characters in the openapi spec method descriptions((#8597)[https://github.com/ZcashFoundation/zebra/pull/8597])
+- Custom testnets don't support custom HRPs anymore.
```

Comment:
They didn't prior to this change either, it was never added to the config deserialization.

```suggestion
```